third-party/chromium/data/verify_signed_data/annotate_test_data.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
```

Comment:
I originally kept this script in the webpki repo (inherited from Chromum's tree) because I thought we might make our own tests using the same format. But instead of using that format, you made your own format to extend the tests. That is OK, but if we're not going to add our own tests using the Chromium format then we should just avoid adding annotate_test_data.py here.

Also, if we're going to use a different format for new tests, we should probably follow up this PR with another PR that converts all the Chromium test vectors into our new preferred format, so that we don't have two formats. (I don't have strong feelings about using the Chromium format or not. I originally used the Chromium tests because I was in a hurry to get tests of any kind written, and because I thought I might some day reuse their entire test suite in webpki. But that's kind of outdated thinking at this point.)